pymtl3/passes/CollectSignalPass.py

```diff
@@ -0,0 +1,131 @@
+"""
+========================================================================
+CollectSignalPass.py
+========================================================================
+#collects signals and stored it as _collectsignals attribute of top.
```

Comment:
The lines included inside `"""` is treated as if they are multi-line comments so the `#` at the beginning is not necessary.

Reply:
Also as a general question, how do I make changes to this pull request? If I change the branch, would changes also carried in this pull request?

Reply:
You can keep pushing commits to this branch, and the changes will be updated here

Reply:
Got it!